CodeMaid/UI/Dialogs/Options/Cleaning/CleaningUpdateDataTemplate.xaml

```diff
@@ -63,6 +64,42 @@
                             <TextBox Style="{DynamicResource TextBoxMultiLine}" Text="{Binding UpdateFileHeaderTypeScript, UpdateSourceTrigger=PropertyChanged}" />
                         </TabItem>
                     </TabControl>
+                    <GroupBox Header="Variables">
+                        <StackPanel Orientation="Horizontal">
+                            <Button Name="UserLogin" Content="$USER_LOGIN$" Width="Auto" Margin="2, 2, 2, 2" Command="{Binding AddHeaderVariableCommand}">
+                                <Button.CommandParameter>
```

Comment:
I think these bindings will be simplified after removing the View code from the ViewModel.